iconservice/icx/issue/issue_regulator.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ICON Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Tuple
+
+from .regulator_storage import RegulatorStorage
+from ...database.db import ContextDatabase
+from ...iconscore.icon_score_context import IconScoreContext
+
+
+# todo: implement fee related logic
+class IssueRegulator:
+    _MAX_I_SCORE = 1_000
```

Comment:
This seems to mean the icx and I-Score exchange rate. Modify name

Reply:
I have changed the name: _I_SCORE_EXCHANGE_RATE